smithy-model/src/main/java/software/amazon/smithy/model/loader/IdlModelParser.java

```diff
@@ -797,22 +797,29 @@ private void parseMembers(LoadOperation.DefineShape op, MemberParsing memberPars
 
         while (!eof()) {
             if (peek() == '}') {
+                clearPendingDocs();
```

Comment:
nit: I think would be easier to read if it was moved before the `expect('}')` at the end of the method after the break.